docs/releasenotes/4.0.0.rst

```diff
@@ -307,3 +307,15 @@ Bugs
     # robotidy: off
     # robotidy: on
     *** Settings ***
+
+- non templated tests in suites with ``Test Template`` set will no longer be aligned together with templated tests by
```

Comment:
It's hard for me to understand this sentence :D 
I mean, it's correct, but just hard to get all the meaning out of it.

Reply:
Yeah.. it's hard to write release notes. Maybe ``Non templated tests will no longer be aligned with templated tests by..`` ?

Reply:
Maybe `Test cases with no template will no longer be aligned together with other tests in a templated test case file.`?

Reply:
I like it, I will go with it